packages/svelte/src/compiler/phases/2-analyze/css/css-analyze.js

```diff
@@ -57,10 +57,12 @@ const analysis_visitors = {
 			node.selectors.length >= 1 &&
 			node.selectors[0].type === 'PseudoClassSelector' &&
 			node.selectors[0].name === 'global' &&
-			node.selectors.every(
-				(selector) =>
-					selector.type === 'PseudoClassSelector' || selector.type === 'PseudoElementSelector'
-			);
+			// we want :global(...) and :global or :global.x, but not div:global
```

Comment:
is this comment right? i thought `div:global` was allowed now?

Reply:
It is allowed, but only if the whole selector is global it is marked as such. I'll enhance the comment